src/test/java/at/medunigraz/imi/bst/conceptextraction/ConceptExtractionTest.java

```diff
@@ -43,12 +43,15 @@ public void preferredTerm() throws UnirestException {
 
     @Test
     public void addSynonyms() throws UnirestException {
-        Assert.assertThat(GraphUtils.addSynonymsFromBestConceptMatch("cholangiocarcinoma"),
```

Comment:
@michelole what's the rationale for modifying this test? It precisely tracks that nothing changed in the API since it was introduced originally. Please write new independent tests if you want to check something else.

Reply:
I probably had changed the test in the past (for testing!) and did not realize it. I reverted the commit.